osaca/parser/register.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+from osaca.parser.operand import Operand
+
+
+class RegisterOperand(Operand):
+    def __init__(
+        self,
+        name=None,
+        width_id=None,
+        prefix_id=None,
+        reg_id=None,
+        regtype_id=None,
```

Comment:
Is it possible that `regtype` is never used anywhere?

Reply:
Removed reg_id and now use regtype https://github.com/RRZE-HPC/OSACA/commit/c2bd4841708deb76492a0054810ab82b7ebea04c